packages/components/src/DataList/components/DataListHeaderTile/components/DataListSortingOptions.css

```diff
@@ -0,0 +1,44 @@
+.optionsList {
```

Comment:
There's a lot of shared styling between this menu and [DataListActionsMenu](https://github.com/GetJobber/atlantis/blob/7c2af0111b3ee141b35861872f7fdf7f1aca3c57/packages/components/src/DataList/components/DataListActionsMenu/DataListActionsMenu.css#L1-L28) - would it make sense to share the code where we have overlap?

Reply:
When you say share the code, do you mean have 1 file that merges `optionsList`/`menu` into one variable?  

It makes sense, but I wonder what that would look like in the code.  For example, I'd probably want to put that css in `DataList.css` to easily share between both files.  Let me know if I'm way off base here but...

I wonder two things:
1. Would we have to move the other variables from `DataListActionsMenu` and `DataListSortingOptions` into `DataList.css` and then delete those other css files and rely on `DataList.css`?  I think the answer is yes, which again I'm ok with - just wondering if we have a restriction on how long css files are.
2. When it comes to the case of `DataListSortingOptions` and `ComboboxOption` having identical `options` css -  in this case, I wonder where we should put the shared css. I don't _think_ we have a component that reaches outside of it's own file to grab css from another component (calling `ComboboxOption` inside `DataList` feels strange?).

I think what this comes down to is the conversation Eric and I are having below - we are now building the same UI for different selection components (that all have slightly different functionality) and eventually we'll need to have a component that fits the needs of `ComboboxContent`/`Sorting`/`ActionsMenu`, etc.  Let me know your thoughts or if I took this in a completely different direction then you intended 😆